File: AFSport.Service/Model/Estoque.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AFSport.Service.Model
{
    public class Estoque
    {
        public Estoque()
        {

        }

        public Estoque(int quantidade, int idProduto, int idUsuario)
        {
            this.Quantidade = quantidade;
            this.IdProduto = idProduto;
            this.IdUsuario = idUsuario;
        }

        public int IdEstoque { get; set; }
        public int Quantidade { get; set; }
        public int IdProduto { get; set; }
        public Produto Produto { get; set; }
        public int IdUsuario { get; set; }
        public Usuario Usuario { get; set; }

        public void Incrementa(int quantidade)
        {
            this.Quantidade += quantidade;
        }

        public void Decrementa(int quantidade)
        {
            this.Quantidade -= quantidade;
        }
    }
}

File: Painel/src/app/estoques/estoque/estoque.ts
import { Produto } from "src/app/produtos/produto/produto";
import { Usuario } from "src/app/usuarios/usuario/usuario";

export interface Estoque {
    idEstoque: number;
    quantidade: number;
    idProduto: number;
    produto: Produto;
    idUsuario: number;
    usuario: Usuario;
    isAtivo: boolean;
}